react/components/ShippingOptions.tsx

```diff
@@ -18,6 +24,22 @@ const ShippingOptions: FunctionComponent<CustomProps> = ({
 
   return (
     <Fragment>
+      <DeliveryOptionsAvailability
+        numberOfItems={numberOfItems}
+        numberOfUnavailableItems={numberOfUnavailableItems}
+      />
+      {numberOfUnavailableItems ? (
```

Comment:
In general we use the `&&` without returning `null` in this scenarios.

Reply:
I was doing this before, but when `numberOfUnavailableItems` was zero, it was showing it (the zero) and we do not want it.

Reply:
What about `numberOfUnavailableItems > 0 &&`?

Reply:
that works. nice!

Reply:
that works. Nice!